app/networkapi/features/views.py

```diff
@@ -0,0 +1,33 @@
+from django.utils import timezone
+from django.db.models import Q
+
+from rest_framework.generics import ListAPIView, RetrieveAPIView
+
+from networkapi.features.models import Feature
+from networkapi.features.serializers import FeatureSerializer
+
+
+class FeatureListView(ListAPIView):
+    """
+    A view that permits a GET to allow listing of Features
+    in the database
+    """
+    now = timezone.now()
+    queryset = Feature.objects.filter(
+        Q(publish_after__lt=now),
```

Comment:
I can make it a kwarg, but I'll have to pass it as the second param. I thought it read better this way.